cmd/miner/main.go

```diff
@@ -87,6 +89,8 @@ var (
 	rpcServer *rpc.Server
 )
 
+const Role = common.Miner
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const Role should have comment or be unexported